$MirOS$

	#1, #2: Fix spelling; suggested for upstream inclusion.
	#3, #4: LYmsec_delay() is needed when USE_MOUSE is
	defined, too; suggested for upstream inclusion.

--- src/LYUtils.c.orig	Mon Feb 13 02:45:53 2012
+++ src/LYUtils.c	Sat Mar  3 18:20:22 2012
@@ -3316,7 +3316,7 @@ void change_sug_filename(char *fname)
     }
 
     /*
-     * Replace all but the last period with _'s, or second to last if last is
+     * Replace all but the last period with _s, or second to last if last is
      * followed by a terminal Z or z, or GZ or gz,
      * e.g., convert foo.tar.Z to foo.tar_Z
      * or, convert foo.tar.gz to foo.tar-gz
@@ -6875,7 +6875,7 @@ void WriteInternalTitle(FILE *fp0, const
 
 /*
  * This is used to start most internal pages, except for special cases where
- * the embedded HREF's in the title differ.
+ * the embedded HREFs in the title differ.
  */
 void BeginInternalPage(FILE *fp0, const char *Title,
 		       const char *HelpURL)
@@ -7620,30 +7620,8 @@ int put_clip(const char *s)
 
 #endif /* __EMX__ */
 
-/*
- * Sleep for a number of milli-sec.
- */
-void LYmsec_delay(unsigned msec)
-{
-#if defined(_WINDOWS)
-    Sleep(msec);
+#define WANT_LYmsec_delay
 
-#elif defined(HAVE_NAPMS)
-    napms((int) msec);
-
-#elif defined(DJGPP) || defined(HAVE_USLEEP)
-    usleep(1000 * msec);
-
-#else
-    struct timeval tv;
-    unsigned long usec = 1000UL * msec;
-
-    tv.tv_sec = usec / 1000000UL;
-    tv.tv_usec = usec % 1000000UL;
-    select(0, NULL, NULL, NULL, &tv);
-#endif
-}
-
 #if defined(WIN_EX)		/* 1997/10/16 (Thu) 20:13:28 */
 
 int put_clip(const char *szBuffer)
@@ -7892,6 +7870,32 @@ void LYCloselog(void)
 }
 
 #endif /* SYSLOG_REQUESTED_URLS */
+
+#if defined(WANT_LYmsec_delay) || defined(USE_MOUSE)
+/*
+ * Sleep for a number of milli-sec.
+ */
+void LYmsec_delay(unsigned msec)
+{
+#if defined(_WINDOWS)
+    Sleep(msec);
+
+#elif defined(HAVE_NAPMS)
+    napms((int) msec);
+
+#elif defined(DJGPP) || defined(HAVE_USLEEP)
+    usleep(1000 * msec);
+
+#else
+    struct timeval tv;
+    unsigned long usec = 1000UL * msec;
+
+    tv.tv_sec = usec / 1000000UL;
+    tv.tv_usec = usec % 1000000UL;
+    select(0, NULL, NULL, NULL, &tv);
+#endif
+}
+#endif /* WANT_LYmsec_delay || USE_MOUSE */
 
 #if defined(WIN_EX) || defined(__CYGWIN__)	/* 2000/03/07 (Tue) 17:17:46 */
 
